packages/devtools_app/lib/src/screens/debugger/call_stack.dart

```diff
@@ -110,6 +110,8 @@ class _CallStackState extends State<CallStack>
     return isAsyncBreak
         ? result
         : DevToolsTooltip(
+            // False positive.
+            // ignore: avoid-nested-conditional-expressions
```

Comment:
You can move explanation to the right of the directive:

            // ignore: avoid-nested-conditional-expressions; False positive.

Reply:
Done! here and elsewhere